packages/commonwealth/server/routing/router.ts

```diff
@@ -370,6 +372,21 @@ function setupRouter(
     getRelatedCommunitiesHandler.bind(this, serverControllers),
   );
 
+  registerRoute(
+    router,
+    'get',
+    '/communityStakes',
+    getCommunityStakeHandler.bind(this, models, serverControllers),
+  );
```

Comment:
Should use RESTful path: `/communities/:communityId/stake`

Reply:
It can be `…/stakes` actually since it's a list

Reply:
I will update the route naming. But for clarity, since communityStakes, is a separate table from communities, should it be its own top-level route? Like

`/communityStakes/:communityId/:stakeId`

And then the second question is, since the key is (community_id, stake_id), should they both be params in the path?

Reply:
Yeah that works